tools/python_api/src_cpp/include/py_query_result.h

```diff
@@ -56,9 +58,12 @@ class PyQueryResult {
 
     bool getNextArrowChunk(const std::vector<std::unique_ptr<DataTypeInfo>>& typesInfo,
         py::list& batches, std::int64_t chunk_size);
+
     py::object getArrowChunks(
         const std::vector<std::unique_ptr<DataTypeInfo>>& typesInfo, std::int64_t chunkSize);
 
+    static py::object convertRdfVariantToPyObject(const Value& value);
```

Comment:
Seems that you can hide this in cpp file. If so, you don't have to include Value in header anymore.